server/routers/project.userfield.router.js

```diff
@@ -3,35 +3,39 @@ const getDB = require ('../database/bootstrap.database.js');
 
 const projectUserFieldRouter = express.Router();
 
-projectUserFieldRouter.post('project/create/userfield', (req, res) => {
+projectUserFieldRouter.post('project/create/userfieldid', (req, res) => {
+    const projectid = req.params.projectid;
     const { targetDemoData, skillData, descriptionData } = req.body;
     const db = getDB();
-    db.create_project_userfield( [targetDemoData, skillData, descriptionData] )
+    db.create_project_userfield( [projectid, targetDemoData, skillData, descriptionData] )
         .then( promise => res.send())
         .catch( err => res.send(err));
 })
 
 projectUserFieldRouter.get('project/userfield/:userfieldid', (req, res) => {
+    const projectid = req.params.projectid;
     const userfieldid = req.params.userfieldid;
     const db = getDB();
-    db.find_project_userfield( [userfieldid] )
-        .then( userfield => res.send(userfield) )
+    db.find_project_userfield( [projectid, userfieldid] )
+        .then( userfield => res.send(userfieldid) )
```

Comment:
As of right now the RES is sending a number that it received from the REQ (`userfieldid`)
The RES should just send the entire promise it receives from the DB

My Solution:
`.then( userfield => res.send( userfield ))`

Let me know what you think.